ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/visitor/core/entities/classes/ClassIdentity.java

```diff
@@ -0,0 +1,34 @@
+package edu.usb.argos.ASTProcessor.visitor.core.entities.classes;
+
+import java.util.List;
+
+public class ClassIdentity {
+
+    private final String name;
+    private final String packageName;
+    private final List<String> modifiers;
+    private final List<AnnotationInfo> annotations;
+
+    public ClassIdentity(String name, String packageName, List<String> modifiers, List<AnnotationInfo> annotations) {
```

Comment:
use the Lombok annotation to generate the constructor with all parameters, and the same thing with other entities

Reply:
Done